packages/types/src/api.ts

```diff
@@ -19,3 +19,11 @@ export interface ClerkPaginationParams {
   limit?: number;
   offset?: number;
 }
+
+/**
+ * Pagination params
+ */
+export interface ClerkPaginationResponse<T> {
```

Comment:
🙃 
```suggestion
export interface ClerkPaginatedResponse<T> {
```

Reply:
I was thinking the same but then I decided to follow the pattern of `ClerkPaginationParams`. I could go back to `ClerkPaginatedResponse`